drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -728,9 +728,9 @@ def _add_msg_if_exists(ids_list, message_template, message_class):
 
         if self.weight > 0:
             if self.attempts_remain:
-                grade_feedback_template = FeedbackMessages.GRADE_FEEDBACK_TPL
+                grade_feedback_template = self.i18n_service.gettext(FeedbackMessages.GRADE_FEEDBACK_TPL)
```

Comment:
I don't understand this fix: you've added a call to `gettext`, but the `grade_feedback_template` variable is passed to `gettext` just a few lines below.  Are you sure this change fixes the problem?